src/Avalonia.Controls/Grid/GridLinesRenderer.cs

```diff
@@ -0,0 +1,95 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using Avalonia.VisualTree;
+using Avalonia.Media;
+
+namespace Avalonia.Controls
+{
+    internal class GridLinesRenderer : Control
+    {
+        /// <summary>
+        /// Static initialization
+        /// </summary>
+        static GridLinesRenderer()
+        {
+            var oddDashArray = new List<double>();
+            oddDashArray.Add(_dashLength);
```

Comment:
even and odd are the same

Reply:
Nice catch! Double-checked the original source, i've mangled it while copypasting it seems